packaging/CMakeLists.txt

```diff
@@ -45,6 +45,19 @@ foreach (dep IN ITEMS Halide_LLVM Halide_wabt)
     endif ()
 endforeach ()
 
+##
+# Python bindings
+##
+
+if (WITH_PYTHON_BINDINGS)
+    set(Halide_INSTALL_PYTHONDIR "${CMAKE_INSTALL_LIBDIR}/python3/site-packages"
+        CACHE STRING "Path to Halide Python bindings folder")
+    install(TARGETS Halide_Python
+            EXPORT Halide_Python
+            LIBRARY DESTINATION ${Halide_INSTALL_PYTHONDIR} COMPONENT Halide_Python
+            NAMELINK_COMPONENT Halide_Python)
```

Comment:
Nothing is done with this `EXPORT` set... either:

1. It should not be exported, or
2. it needs to be added to the set of CMake targets available via `find_package`. This will involve editing `packaging/common/HalideConfig.cmake` to load it when the `Python` component is requested. This will also include calling `find_dependency` on the correct version of Python (which will need to be recorded).

In the case of (2), testing should be added. In the case of (1), an issue should be opened.

Reply:
I'd say just (1) is more than enough for a start :)

Reply:
Not exported, issue opened.